src/common/components/repositories-list/index.js

```diff
@@ -62,57 +64,51 @@ class RepositoriesList extends Component {
     }
   }
 
-  onPageLinkClick(pageNumber) {
-    this.props.dispatch(fetchUserRepositories(pageNumber));
-  }
-
   render() {
-    const isLoading = this.props.repositories.isFetching;
-    const pageLinks = this.props.repositories.pageLinks;
+    const isLoading = this.props.snaps.isFetching;
 
     return (
       <div>
-        { this.props.repositories.success && pageLinks &&
-          <div className={ styles['page-links-container'] }>
-            <PageLinks { ...pageLinks } onClick={ this.onPageLinkClick.bind(this) } />
-          </div>
-        }
         { isLoading &&
           <div className={ spinnerStyles }>
             <Spinner />
           </div>
         }
-        { this.props.repositories.success &&
-          this.props.repositories.repos.map(this.renderRepository.bind(this))
-        }
-        { this.props.repositories.success && pageLinks &&
-          <div className={ styles['page-links-container'] }>
-            <PageLinks { ...pageLinks } onClick={ this.onPageLinkClick.bind(this) } />
-          </div>
-        }
+        <Table>
+          <THead>
+            <TR>
+              <TH>Name</TH>
+              <TH>Configured</TH>
+              <TH>Registered for publishing</TH>
+              <TH>Latest build</TH>
+            </TR>
+          </THead>
+          <TBody>
+            { this.props.snaps.success &&
+              this.props.snaps.snaps.map(this.renderRow.bind(this))
+            }
+          </TBody>
+        </Table>
```

Comment:
Thanks for pointing this out. I'll re-write this based on the interactive table from MAAS. I'm surprised this isn't in Vanilla.